### ISSUE TITLE
gulp build failing

### ISSUE THREAD

**Issue by evillemez** [mgcrea/angular-strap#1834]
I feel like I must be overlooking something basic - but I can't get the project to build.  I've been googling for a bit, no luck yet.  This is after a fresh clone, npm/bower install.  Globally updated gulp as the readme describes.

```
gulp build
Found 1 undertaker-registry to load with ng-factory
Using ng-factory config file ~/code/angular-strap/ngfactory.json

/Users/evillemez/code/angular-strap/node_modules/gulp/node_modules/undertaker/lib/helpers/validateRegistry.js:34
      throw err;
            ^
AssertionError: Custom registry must have `init` function
    at validateRegistry (/Users/evillemez/code/angular-strap/node_modules/gulp/node_modules/undertaker/lib/helpers/validateRegistry.js:28:5)
    at Gulp.registry (/Users/evillemez/code/angular-strap/node_modules/gulp/node_modules/undertaker/lib/registry.js:18:3)
    at ngFactory (/Users/evillemez/code/angular-strap/node_modules/ng-factory/lib/index.js:37:9)
    at Function.ngFactory.use (/Users/evillemez/code/angular-strap/node_modules/ng-factory/lib/index.js:60:37)
    at Object.<anonymous> (/Users/evillemez/code/angular-strap/gulpfile.js:5:36)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
```

**Comment by mgcrea** [comment]
You're using a gulp3 global instance to run a gulp4 local project. `$(npm bin)/gulp` should fix it.

**Comment by evillemez** [comment]
Even running `node_modules/.bin/gulp build` yields the same error after a fresh install... there's got to be something else factoring in to this, I'll keep digging.

**Comment by logorrhea** [comment]
@evillemez I'm having the same issue. Let me know if you find a solution!

**Comment by evillemez** [comment]
@logorrhea I didn't find a way around it, no... I ended up manually making the change I needed in both src & dist.

@mgcrea Maybe this should be reopened - looks like other builds are starting to fail as well.

**Comment by benshell** [comment]
I just had this problem too, and found the issue. ngFactory hasn't had a release in awhile, and it's loading undertaker-registry 0.0.2 instead of the needed 0.0.3. See https://github.com/ng-tools/ng-factory/blob/b58941386e67e7e11fb670c831ab47320841592e/package.json

You can switch to the HEAD version of ngFactory (which uses 0.0.3) by changing in it package.json, e.g.:
"ng-factory": "https://github.com/ng-tools/ng-factory/tarball/master"

But that said, now I'm getting a different error when trying to build:
[15:17:09] Error: Cannot find module 'debug-utils'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/Users/Ben/Streamline/angular-strap/node_modules/factory-angular-channels/lib/channels/index/helpers/bower.js:13:1)
...

**Comment by mgcrea** [comment]
Just released a new `v1.0.11`. Hope it fixes your issues.
